docs/specification.md

```diff
@@ -97,8 +97,9 @@ compromised, new metadata is issued to revoke its trust.
 
 Second, TUF allows for defining _namespaces_ for the repository. TUF's notion of
 namespaces aligns with Git's, and TUF namespaces can be used to reason about
-both Git refs and files tracked within the repository. Namespaces are combined
-with TUF's _delegations_ to define sets of actors who are authorized to make
+both Git refs (tags and branches) and files tracked within the repository. 
```

Comment:
I've defined git refs at the start of the document. The reason I don't want to specify this here is because people may have different interpretations of "branches". Some may consider only `refs/heads/*` as branches while others may consider anything under `refs/*` that is NOT `refs/tags/*` as branches. We use the RSL to track `refs/gittuf/policy`, for example. The definition describes that arbitrary refs may exist.

Reply:
@JustinCappos can I reverse this specific change?

Reply:
Yes, this is fine.